README.md

```diff
@@ -4,26 +4,23 @@ This guide explains how to use a React design system starter powered by:
 
 - 🏎 [Turborepo](https://turbo.build/repo) — High-performance build system for Monorepos
 - 🚀 [React](https://reactjs.org/) — JavaScript library for user interfaces
-- 🛠 [Tsup](https://github.com/egoist/tsup) — TypeScript bundler powered by esbuild
-- 📖 [Storybook](https://storybook.js.org/) — UI component environment powered by Vite
+- 🌈 [TailwindCSS](https://tailwindcss.com/) - A utility-first CSS framework
+- 🧑‍🚀 [Astro](https://astro.build/) - All-in-one web framework for building fast, content-focused websites
 
 As well as a few others tools preconfigured:
 
-- [TypeScript](https://www.typescriptlang.org/) for static type checking
 - [ESLint](https://eslint.org/) for code linting
 - [Prettier](https://prettier.io) for code formatting
 - [Changesets](https://github.com/changesets/changesets) for managing versioning and changelogs
-- [GitHub Actions](https://github.com/changesets/action) for fully automated package publishing
+- [PNPm](https://pnpm.io/) for managing dependency packages
 
-## Using this example
+## Installation Guide
 
-Clone the design system example locally or [from GitHub](https://github.com/vercel/turbo/tree/main/examples/design-system):
+Clone the design system locally
```

Comment:
can we add codeblock  below with ready to use command?
```
git clone git@github.com:Selleo/design-system.git
```

Reply:
Added to other bash steps
```bash
git clone git@github.com:Selleo/design-system.git
cd design-system
pnpm install
```